tools/loader/Loader.cpp

```diff
@@ -516,6 +516,8 @@ void Loader::compile(CompilationContext &cctx) {
     for (auto function : module->getFunctions()) {
       std::string filename =
           function->getFilename() + "_" + dumpGraphDAGFileOpt;
+      if (module->getFunctions().size() == 1)
```

Comment:
nit: Our style is generally to always use curly braces even on single line ifs.

Reply:
Done.